api_admin/spec.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html>
 <head>
-    <title>Polygon ID - Self-hosted Admin Platform</title>
+    <title>Polygon ID - Issuer UI API</title>
```

Comment:
This is the issuer API right? I would remove "UI" here since we use it for a specific part of the stack, not in its generic sense.